### PR TITLE
Introducing resources chapter for resource-specific rules

### PR DESCRIPTION
**Pull request by fmueller**
Some editorial changes, suggested by @tfrauenstein 

All resource-related rules are grouped in own chapter.

### CODE REVIEW

File: resources/Resources.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    /carts/1681e6b88ec1/cart-items/1
+
+However, if the resource can be accessed directly via its unique id, then the API should expose it as a top-level resource. For example, customer is a collection for sales orders; however, sales orders have globally unique id and some services may choose to access the orders directly, for instance:
+
+    /customer/1681e6b88ec1
+    /sales-order/5273gh3k525a
+
+## {{ book.should }} Limit of Resources
+
+To keep maintenance manageable, and to avoid violations of the “separation of concern” principle, an API should not expose more than 16 resources. If you exceed 16 resources, first check if you can split them into separate subdomains with distinct APIs.
```
Comment (whiskeysierra) [inline resources/Resources.md]: I haven't seen that rule before. It's kind of a random magic number, isn't it? Why 16?

Comment (fmueller) [inline resources/Resources.md]: Right, this sounds strange. Any historical background on this by @tfrauenstein @mostlymagic or @ePaul?

Comment (ePaul) [inline resources/Resources.md]: I don't remember having seen this (nor the next) rule before.

Looking into the history, this was introduced in commit a68803b1ed ("Merging all pending changes from the internal Google doc") from you. I suppose someone proposed it in the Google doc and it was then simply copied here?

Comment (whiskeysierra) [inline resources/Resources.md]: > I suppose someone proposed it in the Google doc and it was then simply copied here?

That makes sense.

Comment (fmueller) [inline resources/Resources.md]: Ok, then I'll drop it.

**Comment by whiskeysierra** [comment]
I created #28 for the discussion/change about the limit.

File: resources/Resources.md (hunk 1 above)
Comment (whiskeysierra) [inline resources/Resources.md]: > Ok, then I'll drop it. 

Sorry, I saw that too late.

> Nevertheless we should say sth about separation of concerns and single responsibility of APIs. Do we have a rule for this?

Not yet, but :+1:

Comment (mostlymagic) [inline resources/Resources.md]: I agree with the rule in general, but I don't think it makes sense to use a hard number

**Comment by ePaul** [comment]
Parent-Issue: #41.

**Pull request merged**